### PR TITLE
Category result overview clean up

### LINKED ISSUE #605: Category result overview clean up
The lines for missing bonus points should be located exactly under the respective heading.

edit:
The view should be responsive

grafik

### PR DESCRIPTION
**Pull request by djoko1989**
closes #605

### CODE REVIEW

**Review by Zitrone44 — CHANGES_REQUESTED** [review]
Looks good. Only the changes in the `results` component that add comments should be removed and added in another pull request if necessary. Since they are not part of the issue #605.

**Comment by djoko1989** [comment]
I have only worked on 4 issues and don't know the others so I have separated each of my issues into a branch

**Comment by djoko1989** [comment]
here it is just a matter of sorting out the results of the categories

**Comment by djoko1989** [comment]
I think, now all was removed

**Review by Zitrone44 — DISMISSED** [review]

File: web-gui/src/app/page-components/results/results.component.ts [hunk 1]
```diff
@@ -23,6 +29,24 @@ export class ResultsComponent {
...
+  this.postComment.push(this.comment);
+
+  this.comment = "";
+  }
+  }
+
+  delete(){
+
+    this.postComment.pop();
+  }
```
Comment (Zitrone44) [inline web-gui/src/app/page-components/results/results.component.ts:47]:
```suggestion
```
changes in this file should also be reverted.

**Review by Zitrone44 — CHANGES_REQUESTED** [review]

**Review by Zitrone44 — APPROVED** [review]

**Pull request closed without merging**